### PR TITLE
Vikram readline bottom

### PR DESCRIPTION
**Pull request by vikrambombhi**
Fixed the bug where the message you were typing got split if you received a message half way though. Also added a newline when receiving a new message to fix styling but its still kinda fucked, Hao please halp with that.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -47,13 +47,15 @@ socket.on('chat', (data) => {
     const date = new Date(data.date)
     const hour = date.getHours()
     const min = date.getMinutes()
+	console.log('\n')
```
Comment (JackyChiu) [inline app.js:50]: What is this extra end line for? 
(`console.log()` already produces a end line)

File: input.js [hunk 2]
```diff
@@ -32,8 +32,12 @@ const Input = (() => {
         })
     }
 
+	function resetCursor(){
+		readline.cursorTo(process.stdout, 0)
+		rl.prompt(true)
```
Comment (JackyChiu) [inline input.js:37]: Can you explain what this prompt does?

**Review by JackyChiu — COMMENTED** [review]
Can't wait to see this in action 😍

**Comment by JackyChiu** [comment]
Fixes #3

**Comment by BenEmdon** [comment]
may the `--force` be with you to rebase these conflicts

**Comment by JackyChiu** [comment]
RIP

**Pull request closed without merging**